### PR TITLE
Generate ext/spl class synopses from stubs - part 3

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/spl/regexiterator.xml [hunk 1]
```diff
@@ -165,6 +172,20 @@
...
 
+  <section xml:id="regexiterator.props">
+   &reftitle.properties;
+   <variablelist>
+    <varlistentry xml:id="regexiterator.props.replacement">
+     <term>
+      <varname>replacement</varname>
+     </term>
+     <listitem>
+      <para></para>
```
Comment (kocsismate) [inline reference/spl/regexiterator.xml:183]: what should be the description?

File: reference/spl/runtimeexception.xml [hunk 2]
```diff
@@ -25,31 +20,30 @@
...
+    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.exception')/db:partintro/db:section/db:classsynopsis/db:fieldsynopsis[preceding-sibling::db:classsynopsisinfo[1][@role='comment' and text()='&Properties;']]))">
+     <xi:fallback/>
+    </xi:include>
 
-<!-- Comment this section out if the exception does not contain any methods --><!--
-    <classsynopsisinfo role="comment">&Methods;</classsynopsisinfo>
-    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.runtimeexception')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[1])" />
--->
-   <classsynopsisinfo role="comment">&InheritedMethods;</classsynopsisinfo>
-    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.exception')/db:refentry/db:refsect1[@role='description']/descendant::db:constructorsynopsis[1])"><xi:fallback /></xi:include>
```
Comment (kocsismate) [inline reference/spl/runtimeexception.xml:51]: inherited constructors are usually not listed here as far as I remember. Should we consistently do this, or is it ok to get rid of this one?

File: reference/spl/regexiterator.xml (hunk 1 above)
Comment (cmb69) [reply reference/spl/regexiterator.xml:183]: From what I can tell, the replacement is only relavant for `RegexIterator::REPLACE` for which the documentation states:

> Replace the current entry (see preg_replace(); Not fully implemented yet)

So maybe leave the description empty for now.

File: reference/spl/runtimeexception.xml (hunk 2 above)
Comment (cmb69) [reply reference/spl/runtimeexception.xml:51]: It's probably okay to just drop this one for now.

File: reference/spl/splfixedarray.xml [hunk 3]
```diff
@@ -24,36 +24,68 @@
...
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>8.1.0</entry>
+       <entry>
+        <classname>SplFixedArray</classname> implements
+        <interfacename>JsonSerializable</interfacename> and
+        <interfacename>IteratorAggregate</interfacename> now.
+        <interfacename>Iterator</interfacename> is no longer implemented.
```
Comment (cmb69) [inline reference/spl/splfixedarray.xml:81]: Maybe:
```suggestion
        Previously, only <interfacename>Iterator</interfacename> was implemented instead.
```

**Review by cmb69 — COMMENTED** [review]
Thank you!

**Pull request merged**